packages/dropdown/src/__stories__/getKnobs.js

```diff
@@ -28,9 +30,16 @@ export default function getKnobs(props) {
     required = "",
     multiple = false,
     options = [],
+    value,
     ...otherProps
   } = props;
 
+  /**
+   * We're using `text` instead of `object` so that we can provide `undefined`
```

Comment:
Should `object` be used for every knob option that is required in the library?

It's difficult for me to derive the difference between the two from this comment.

Reply:
This is just a workaround for this specific case. The kind of knob that's used is primarily determined by the respective prop's data type. However, choose whatever has the best UX for control. The expected knob to use here would be `object`, but there's an issue with how it doesn't interpret empty values as `undefined`. So we used `text` here instead, with our own logic for empty values.